specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2018-07-10/service.json

```diff
@@ -17187,6 +18191,34 @@
       },
       "x-ms-discriminator-value": "InMage"
     },
+    "RecoveryPlanInMageRcmFailbackFailoverInput": {
+      "description": "Recovery plan InMageRcmFailback failover input.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/RecoveryPlanProviderSpecificFailoverInput"
+        }
+      ],
+      "properties": {
+        "recoveryPointType": {
+          "description": "The recovery point type.",
+          "enum": [
+            "ApplicationConsistent",
+            "CrashConsistent"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "InMageRcmFailbackRecoveryPointType",
+            "modelAsString": true
+          }
+        },
+        "useMultiVmSyncPoint": {
+          "description": "A value indicating whether multi VM sync enabled VMs should use multi VM sync points for failover.",
+          "type": "string"
```

Comment:
If this is contstrained to certain value please add the potential "enum" values with modelAsString: true

Reply:
this field can have true/false value only. It is modelled like other bool values in the spec.

Reply:
A string whose only values are "true" or "false" should at the very least be a string enum.  If you are going to do this it should have been a boolean to begin with but a string enum will allow you to remain consistent with the other places that have a similar property while making it easy for users to use the SDK/API